From aeec046fe995d177f9f31af9278777515f0b0869 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Thu, 27 Aug 2015 15:47:12 +0800
Subject: [PATCH] MLK-11432 ASoC: fsl_sai: ALSA core will restore pin setting
 for cpu_dai when resume back

Revert "MLK-11410-1 ASoC: fsl_sai: restore pin setting for SAI"

This reverts commit 28fd246b81d9cb83c8b31edd829b53328cb46729.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 sound/soc/fsl/fsl_sai.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/sound/soc/fsl/fsl_sai.c b/sound/soc/fsl/fsl_sai.c
index 6cdfed7..c2cca34 100644
--- a/sound/soc/fsl/fsl_sai.c
+++ b/sound/soc/fsl/fsl_sai.c
@@ -875,7 +875,6 @@ static int fsl_sai_suspend(struct device *dev)
 	regcache_cache_only(sai->regmap, true);
 	regcache_mark_dirty(sai->regmap);
 
-	pinctrl_pm_select_sleep_state(dev);
 	return 0;
 }
 
@@ -883,8 +882,6 @@ static int fsl_sai_resume(struct device *dev)
 {
 	struct fsl_sai *sai = dev_get_drvdata(dev);
 
-	pinctrl_pm_select_default_state(dev);
-
 	regcache_cache_only(sai->regmap, false);
 	regmap_write(sai->regmap, FSL_SAI_TCSR, FSL_SAI_CSR_SR);
 	regmap_write(sai->regmap, FSL_SAI_RCSR, FSL_SAI_CSR_SR);
-- 
1.8.0

